---
# Documentation: https://wowchemy.com/docs/managing-content/

title: 'Critical assessment of genome-scale metabolic networks: the need for a unified
  standard'
subtitle: ''
summary: ''
authors:
- Aarthi Ravikrishnan
- Karthik Raman
tags:
- '"cobra constraint-based_modelling flux-analysis myown sbml"'
categories: []
date: '2015-02-01'
lastmod: 2020-10-31T20:24:45+05:30
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2020-10-31T15:04:37.704890Z'
publication_types:
- '2'
abstract: Genome-scale metabolic networks have been reconstructed for several organisms.
  These metabolic networks provide detailed information about the metabolism inside
  the cells, coupled with the genomic, proteomic and thermodynamic information. These
  networks are widely simulated using 'constraint-based' modelling techniques and
  find applications ranging from strain improvement for metabolic engineering to prediction
  of drug targets in pathogenic organisms. Components of these metabolic networks
  are represented in multiple file formats and also using different markup languages,
  with varying levels of annotations; this leads to inconsistencies and increases
  the complexities in comparing and analysing reconstructions on multiple platforms.
  In this work, we critically examine nearly 100 published genome-scale metabolic
  networks and their corresponding constraint-based models and discuss various issues
  with respect to model quality. One of the major concerns is the lack of annotations
  using standard identifiers that can uniquely describe several components such as
  metabolites, genes, proteins and reactions. We also find that many models do not
  have complete information regarding constraints on reactions fluxes and objective
  functions for carrying out simulations. Overall, our analysis highlights the need
  for a widely acceptable standard for representing constraint-based models. A rigorous
  standard can help in streamlining the process of reconstruction and improve the
  quality of reconstructed metabolic models.
publication: '*Briefings in Bioinformatics*'
url_pdf: http://dx.doi.org/10.1093/bib/bbv003
doi: 10.1093/bib/bbv003
---
